dashboard/src/app/dashboard/[dashboardId]/layout.tsx

```diff
@@ -44,28 +47,30 @@ export default async function DashboardLayout({ children, params }: DashboardLay
 
   return (
     <DashboardProvider>
-      <section>
-        <BATopbar />
-        <SidebarProvider>
-          <BASidebar dashboardId={dashboardId} />
-          <BAMobileSidebarTrigger />
-          <main className='bg-background w-full overflow-x-hidden'>
-            <ScrollReset />
-            {billingEnabled && (
-              <Suspense fallback={null}>
-                <UsageUpgradeBanner billingDataPromise={getUserBillingData()} />
-              </Suspense>
-            )}
-            <div className='flex w-full justify-center'>{children}</div>
-          </main>
-          {/* Conditionally render tracking script based on server-side feature flag */}
-          {shouldEnableTracking && siteId && <TrackingScript siteId={siteId} />}
+      <DictionaryProvider dictionary={dictionary} initialLanguage={language}>
```

Comment:
Suggestion: Instead of wrapping it in here and in `dashboards/layout.tsx`, this should be added in `dashboard\src\app\Providers.tsx`. That would wrap the entire application in the `DictionaryProvider` 😄